tests/platforms/linux/appimage/test_mixin.py

```diff
@@ -159,7 +159,7 @@ def test_verify_non_linux_docker(tmp_path, first_app_config):
         base_path=tmp_path / "base_path",
         data_path=tmp_path / "briefcase",
     )
-    command.tools.host_os = "WeirdOS"
```

Comment:
As above; It's not being picked up by coverage, but both Darwin and WeirdOS should be tested here.